content/en/blog/falcosidekick-2-28-0/index.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Falcosidekick 2.28.0
+linktitle: Falcosidekick 2.28.0
+author: Thomas Labarussias
+date: 2023-07-28
+slug: falcosidekick-2-28-0
+images:
+  - /blog/falcosidekick-2-28-0/images/falcosidekick-featured.png
+tags: ["Falcosidekick","Release"]
+---
+
+It's summer, it's hot, many people are on holiday, but the Falco community is still there. Six months after the previous version of Falcosidekick, version 2.28.0 is officially available.
+
+The number of pulls of the official image from Dockerhub has also just reached 15M, which is just mind-blowing. It took 3 years to reach the first 5M pulls and now it's less than 6 months. Awesome!
+
+![](images/falcosidekick-docker-pulls.png)
+![](images/falcosidekick-docker-pulls-last-year.png)
+
+Once again, thanks to the hard work of the community, we introduced a bunch of new outputs and features, we hope will be useful. You can read the full changelog [here](https://github.com/falcosecurity/falcosidekick/releases/tag/2.28.0).
+
+## New outputs
+
+At every release, the number of integrations of Falcosidekick increases, we reached 58 available outputs now!!! For sure, the threshold of 60 will be crossed soon. 
+
+### Redis
+
+[Redis](https://redis.com) is well known in-memory database, used for years. It's now a possible output of Falcosidekick, thanks to [pandyamarut](https://github.com/pandyamarut).
+
+### Telegram
+
+[Telegram](https://telegram.org/), the instant messaging platform, is more and more used by companies for notifications, thanks to [zufardhiyaulhaq](https://github.com/zufardhiyaulhaq), it can receive Falco alerts too.
+
+![](images/falcosidekick-telegram.png)
+
+### N8N
+
+You want to extend the possibilities or just avoid to develop a script to react to Falco events? Here's come [n8n](https://n8n.io/).
+
+![](images/falcosidekick-n8n.png)
+
+### Grafana OnCall
+
+We met some Grafana maintainers at last Kubecon and we discussed about integrating Falco, with Falcosidekick, in the OnCall project. It's done.
+
+![](images/falcosidekick-grafana-oncall.png)
+
+### OpenObserve
+
+[OpenObserve](https://openobserve.ai/) is a young but promising full stack observability platform.
+
+![](images/falcosidekick-openobserve.png)
+
+
+## New features
+
+Aside new outputs, we introduced very important and useful new features. Let's recap them.
+
+### Use different methods for the Webhook output
+
+The Webhook output since its implementation used the http method `POST`, you now have choice between `POST` and `PUT`, extending the possible API Rest you can use with.
```

Comment:
```suggestion
Since its implementation, the Webhook output has only used the HTTP method `POST`. Now, you can choose between the `POST` and `PUT` methods to send your data, extending the catalog of possible REST APIs to use it with.
```